### PR TITLE
Ignore EtcdCluster Phase

### PR DESCRIPTION
**Pull request by rjtsdl**
- also extend purge interval from 10 secs to 24 hours

### CODE REVIEW

File: pkg/backup/backup.go [hunk 1]
```diff
@@ -160,7 +160,8 @@ func (b *Backup) Run() {
 		}
 
 		for {
-			<-time.After(10 * time.Second)
+			// Check every 24 hours
+			<-time.After(24 * time.Hour)
```
Comment (rite2nikhil) [inline pkg/backup/backup.go:164]: How many blobs we create in a day, can list call have more latency since it will have more work to do to purge

Comment (rjtsdl) [reply pkg/backup/backup.go:164]: In a day, about [48, 52] could be created.
What do you mean more latency, like a month ?

**Review by rite2nikhil — APPROVED** [review]

File: pkg/cluster/cluster.go [hunk 2]
```diff
@@ -162,7 +152,12 @@ func (c *Cluster) setup() error {
 		return errCreatedCluster
 	case spec.ClusterPhaseRunning:
 		shouldCreateCluster = false
-
+	case spec.ClusterPhaseFailed:
```
Comment (qike-ms) [inline pkg/cluster/cluster.go:155]: I suggest we change phase to running here and set shouldCreateCluster to false and continue instead of panic

**Review by qike-ms — CHANGES_REQUESTED** [review]

File: pkg/cluster/cluster.go (hunk 2 above)
Comment (rjtsdl) [reply pkg/cluster/cluster.go:155]: Sounds good, fixed

Comment (rjtsdl) [reply pkg/cluster/cluster.go]: Applied on a Failed cluster, it works.

**Review by qike-ms — COMMENTED** [review]
LGTM

**Pull request merged**